SphericalShell/OpenMC_Input/OpenMC_Model/SS_Material.py

```diff
@@ -0,0 +1,40 @@
+import openmc
+import numpy as np
+
+def alara_element_densities(elelib_fp):  
+    '''
+    Create a dictionary of element names and their corresponding densities using the ALARA element library.
+    
+    inputs:
+        filepath: path to file containing ALARA element library (str)
+        
+    '''
+    with open(elelib_fp) as ALARA_Lib:
+        libLines = ALARA_Lib.readlines()
+    num_lines = len(libLines)
+    density_dict = {}
+    line_num = 0
+    while line_num < num_lines:
+        element_data = libLines[line_num].strip().split()
+        element_name = element_data[0].lower()
+        density_dict[element_name] = float(element_data[3])
+        line_num += int(element_data[4]) + 1
+    return density_dict
+
+def make_element(element, density_dict, inner_radius, thickness):
+    '''
+    inputs:
+        element: elemental symbol of chosen element (str)
+        density_dict: dictionary with key = elemental symbol & value = density [g/cm^3]

```

Comment:
The volume is used for depletion and during post-processing, where flux data is normalized by volume. Calculating the volume separately for both of these might be best for separation of concerns...I think I was trying to avoid having to import the thickness/radius on each occasion